compose.yml

```diff
@@ -1,12 +1,24 @@
 ---
 # This playbook composes OSTree
+
+- include_vars:
+    file: vars/packages.yml
+
 - name: Clone Fedora-Atomic buildscripts
   git:
     repo=https://pagure.io/fedora-atomic.git
     version=f25
     depth=1
     dest=/workspace/fedora-atomic
 
+- name: Customize Packages
+  lineinfile:
```

Comment:
@gbraad Do you mean simply creating a copy of the json file and edit that? 
If you meant this, +1 from my side.